spec/factories/quiz_participations.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :quiz_participation do
```

Comment:
ditto

Reply:
Apu, I have added them in the branch, `4-write-tests-for-models`.